torch/csrc/distributed/rpc/utils.cpp

```diff
@@ -713,6 +713,85 @@ std::vector<at::IValue> readWrappedPayload(
   payload.resize(payload.size() - additionalPayloadSize);
   return tupleElements;
 }
+
+void populateRemoteProfiledEvents(
+    std::vector<torch::autograd::profiler::Event>& profiledEvents,
+    const torch::autograd::profiler::ProfilerConfig& profilingConfig,
+    const std::vector<std::vector<torch::autograd::profiler::Event>>&
+        event_lists) {
+  // Gather all events into a vector
+  for (auto& l : event_lists) {
+    for (auto& e : l) {
+      profiledEvents.push_back(e);
+    }
+  }
+  // find __start_profile event and __cuda_start_event.
+  bool cuda_profiling_enabled =
```

Comment:
inconsistent naming convention

Reply:
Sorry, didn't quite follow. What's the inconsistency in this case?

Reply:
Oh, it should be named `cudaProfilingEnabled`, will update